virtualizarr/xarray.py

```diff
@@ -397,6 +412,18 @@ def to_kerchunk(
 
             return None
         elif format == "parquet":
-            raise NotImplementedError()
+            from kerchunk.df import refs_to_dataframe
+
+            if isinstance(filepath, Path):
+                url = str(filepath)
+            elif isinstance(filepath, str):
+                url = filepath
+
+            return refs_to_dataframe(
```

Comment:
I don't think `refs_to_dataframe` actually has a return value? Is that return there for type hinting reasons?

Also worth clarifying with a comment that `refs_to_dataframe` doesn't return a dataframe, it really should be called `refs_to_parquet`.

Reply:
yeah you are right. I was thinking that it might sometime return a future or something, but that isn't how it works. I'll remove the return